x11-packages/qbittorrent-nox/build.sh

```diff
@@ -0,0 +1,19 @@
+TERMUX_PKG_HOMEPAGE=https://www.qbittorrent.org/
+TERMUX_PKG_DESCRIPTION="A Qt based BitTorrent client"
+TERMUX_PKG_LICENSE="GPL-2.0, GPL-3.0"
+TERMUX_PKG_MAINTAINER="Joshua Kahn @TomJo2000"
+TERMUX_PKG_VERSION="4.6.4"
+TERMUX_PKG_SRCURL=https://github.com/qbittorrent/qBittorrent/archive/refs/tags/release-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=5842eb2cc1745b72b090df562d35a7ec0a6469d898d411544808731578447e1d
+TERMUX_PKG_FORCE_CMAKE=true
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_DEPENDS="libicu, libtorrent-rasterbar, openssl, qt5-qtbase"
+TERMUX_PKG_BUILD_DEPENDS="qt5-qttools, qt5-qtsvg, boost, libc++"
+
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
+-DSTACKTRACE=OFF
+-DGUI=OFF
+-DCMAKE_BUILD_TYPE=Release
+-DBUILD_SHARED_LIBS=off
+-DCMAKE_INSTALL_PREFIX="${TERMUX_PREFIX}"
```

Comment:
This is set automatically by the building system.

https://github.com/termux/termux-packages/blob/8691c1f3a1adaf31ed1e6dc46c3e9fb1f2263ead/scripts/build/configure/termux_step_configure_cmake.sh#L58

Reply:
Not sure how I didn't catch that.

Reply:
> How do you like to squash commits? Let me know the method

Usually `git commit --amend`, followed by `git push --force-with-lease=branchname`
which works great if you just wanna modify the latest commit.

---

If a PR ends up with multiple commits I usually do an "interactive rebase" with `git rebase -i HEAD~42`
(where `42` gets swapped out for the number of commits I actually need to work on)
I can then reorder, squash or "fixup" ***my own*** commits.
But crucially you do not mess with commits already on the master branch.

---

"Fixup" commits are another option if you are a bit more organized than me.
In that case you commit for "fixup" commits using `git commit --fixup=ref`,
where `ref` is the commit you want to squash them into later.
- Example: 
`e08e34581` is the commit you want to fixup, so you can commit your new changes with,
`git commit --fixup=e08e34581`
Which will then generate a commit along the lines of:
```styl
[qbittorrent 585e9e714] fixup! addpkg(x11/qbittorrent-nox): 4.6.4 closes #10081
```
These can be automatically squashed before pushing using `git rebase --autosquash`, this can be combined with an interactive rebase if you like.

Reply:
I still see -DCMAKE_INSTALL_PREFIX not removed...